TESTS/psa/its_ps/main.cpp

```diff
@@ -109,6 +111,26 @@ void pits_ps_test()
 
     status = get_info_func(stype, 5, &info);
     TEST_ASSERT_EQUAL(PSA_ERROR_DOES_NOT_EXIST, status);
+
+    if (stype == its) {
+        return;
+    }
+
+    flags = PSA_STORAGE_FLAG_NO_REPLAY_PROTECTION;
+    status = set_func(stype, 6, TEST_BUFF_SIZE, write_buff, flags);
+    TEST_ASSERT_EQUAL(PSA_SUCCESS, status);
+
+    status = get_info_func(stype, 6, &info);
+    TEST_ASSERT_EQUAL(PSA_SUCCESS, status);
+    TEST_ASSERT_EQUAL(flags, info.flags);
+
+    flags = PSA_STORAGE_FLAG_NO_REPLAY_PROTECTION | PSA_STORAGE_FLAG_NO_CONFIDENTIALITY | PSA_STORAGE_FLAG_WRITE_ONCE;
+    status = set_func(stype, 6, TEST_BUFF_SIZE, write_buff, flags);
+    TEST_ASSERT_EQUAL(PSA_SUCCESS, status);
+
+    status = get_info_func(stype, 6, &info);
+    TEST_ASSERT_EQUAL(PSA_SUCCESS, status);
+    TEST_ASSERT_EQUAL(flags, info.flags);
```

Comment:
How do we know that the KvStore flags are correct in this test? Should we also do an assert to make sure both `KVStore::REQUIRE_REPLAY_PROTECTION_FLAG` and `KVStore::REQUIRE_CONFIDENTIALITY_FLAG` are clear in `kv_info.flags`, to ensure our translation went properly?